CHANGELOG.md

```diff
@@ -4,9 +4,36 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.3.1 (2021-07-05)
+
+This maintenance release contains bugfixes since the 2.3.0 release. We
+deem it moderate priority for upgrading.
```

Comment:
We usually do a summary of the areas covered by the bugfixes

Reply:
For patch releases, I find that redundant since the bugs are listed just below, but I added a blurb about that to the changelog for this release.